docs/pkgdown.yml

```diff
@@ -1,10 +1,10 @@
-pandoc: 2.7.3
-pkgdown: 1.5.1
```

Comment:
@alexanderbates can you please remove all changes to the docs/ folder. I would rather regenerate this myself after merging to master. Otherwise there are so many changes I find it hard to see the wood from the trees. There is however one thing that is missing, you could please edit `_pkgdown.yml` (note leading underscore) to add your new exported function(s). Thanks!

Reply:
Is there an easy way to revert all changes within a specified folder?

Reply:
https://stackoverflow.com/questions/43544919/git-revert-all-committed-files-in-a-directory-to-master